### PR TITLE
Adding a scheduled workflow for stage release

### LINKED ISSUE #106: Add scheduled run for stage tests
We'd like to trigger a stage test run on Wednesday mornings, which is after the AMO stage deploys. 
The documentation on how this should be done can be found here - https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow

### PR DESCRIPTION
**Pull request by AlexandraMoga**
Fix #106

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,6 +29,16 @@ jobs:
       - store_artifacts:
           path: ui-test-release.html
 workflows:
-  build_and_test:
+  commit_workflow:
     jobs:
-        - test
+      - test
+  stage_release_run:
```
Comment (jrbenny35) [inline .circleci/config.yml:35]: Couuld we add some context to this title showing that it is a scheduled job?

**Review by jrbenny35 — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (AlexandraMoga) [reply .circleci/config.yml:35]: So, basically, I should move the `# the workflow runs each Wednesday at 05:00 UTC` comment here?

Comment (jrbenny35) [reply .circleci/config.yml:35]: Not quite, maybe change it to like ```scheduled_stage_release_run```

**Comment by AlexandraMoga** [comment]
I've made the modifications required and merged the PR to make sure that we catch the schedule run tomorrow morning.

**Pull request merged**